terminus-ui/button/src/button.component.spec.ts

```diff
@@ -7,6 +7,7 @@ import {
   tick,
 } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
+import { getDomAttribute } from '@terminus/ngx-tools';
```

Comment:
Oh, also, this is a rebase issue I think.. you can go ahead and remove this line.